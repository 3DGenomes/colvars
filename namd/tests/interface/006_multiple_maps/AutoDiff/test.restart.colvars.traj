# step         multimap               map                    map2                   E_harmonic1          
          20    1.32181390508206e+00   7.88136994459551e-01   2.66838455311252e-01   5.74920474175063e+02
          21    1.32333381440099e+00   7.87084365326980e-01   2.68124724537006e-01   5.72346408416388e+02
          22    1.32499843499643e+00   7.86008754017242e-01   2.69494840489596e-01   5.69533890946580e+02
          23    1.32681180940803e+00   7.84922395373201e-01   2.70944707017415e-01   5.66477924940614e+02
          24    1.32877557243283e+00   7.83833301308302e-01   2.72471135562266e-01   5.63177790203587e+02
          25    1.33088867525256e+00   7.82746035819981e-01   2.74071319716288e-01   5.59637456131597e+02
          26    1.33314745433749e+00   7.81663047324413e-01   2.75742203506540e-01   5.55865397070708e+02
          27    1.33554605510523e+00   7.80585662471148e-01   2.77480196317040e-01   5.51873806107780e+02
          28    1.33807287411328e+00   7.79513447758451e-01   2.79279713177417e-01   5.47684399980809e+02
          29    1.34071006914232e+00   7.78445689233423e-01   2.81132189954447e-01   5.43329016162910e+02
          30    1.34343242971970e+00   7.77380602096209e-01   2.83025913811747e-01   5.38851217929716e+02
          31    1.34621265930532e+00   7.76316933528886e-01   2.84947862888215e-01   5.34297358565785e+02
          32    1.34902315734536e+00   7.75253602708452e-01   2.86884777318454e-01   5.29713562090754e+02
          33    1.35183826184705e+00   7.74189860151053e-01   2.88824200847999e-01   5.25142048506814e+02
          34    1.35463932124803e+00   7.73123014003399e-01   2.90758153622317e-01   5.20613007098999e+02
          35    1.35741399693370e+00   7.72049585724153e-01   2.92682205604773e-01   5.16145964170904e+02
          36    1.36015191430312e+00   7.70969409776841e-01   2.94591252263138e-01   5.11756965962457e+02
          37    1.36284844752086e+00   7.69882621269089e-01   2.96482913125885e-01   5.07452626033224e+02
          38    1.36550633377150e+00   7.68789799122227e-01   2.98358267324635e-01   5.03227765605108e+02
          39    1.36813616323915e+00   7.67687297851399e-01   3.00224432693875e-01   4.99064885257678e+02
          40    1.37074503648023e+00   7.66575883577770e-01   3.02084576451228e-01   4.94952261392840e+02
