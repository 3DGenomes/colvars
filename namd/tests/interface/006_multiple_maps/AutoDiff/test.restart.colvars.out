colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  https://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: Opening trajectory file "test.restart.colvars.traj".
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "multimap"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "map"
colvars:     # componentCoeff = 0.0034027
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "map"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "maprot"
colvars:     # componentCoeff = 0.0068054
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "map2"
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.02
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "map"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "map"
colvars:     # componentCoeff = 0.0034027
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "map"
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "map2"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "map2"
colvars:     # componentCoeff = 0.0034027
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "map2"
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 3 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { multimap }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 1.3 }
colvars:   # targetCenters = { 2 }
colvars:   # targetNumSteps = 20
colvars:   # targetNumStages = 0 [default]
colvars:   # outputAccumulatedWork = off [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 1
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "multimap" will be rescaled to 2500 according to the specified width (0.02).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (2 grid objects in total).
colvars: ----------------------------------------------------------------------
colvars: Loading state from file "test.colvars.state".
colvars:   Restarting collective variable "multimap" from value: 1.32181
colvars:   Restarting collective variable "map" from value: 0.788137
colvars:   Restarting collective variable "map2" from value: 0.266838
colvars:   Restarting harmonic bias "harmonic1" from step number 20.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (2 grid objects in total).
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Saving collective variables state to "test.restart.colvars.state".
